src/bootstrap/bootstrap.py

```diff
@@ -714,7 +714,6 @@ def build_bootstrap(self):
         # See also: <https://github.com/rust-lang/rust/issues/70208>.
         if "CARGO_BUILD_TARGET" in env:
             del env["CARGO_BUILD_TARGET"]
-        env["RUSTC_BOOTSTRAP"] = '1'
```

Comment:
Huh, I always thought this was setting bootstrap for compiling rustc itself. I guess `bootstrap/lib.rs` also sets this variable then?

Reply:
Yes, well, in a bunch of places, but rustbuild sets the variable as needed. Probably even more than strictly necessary, but that's a separate battle.